app/src/main/java/cm/aptoide/pt/promotions/PromotionViewApp.java

```diff
@@ -106,6 +106,10 @@ public boolean isClaimed() {
     return isClaimed;
   }
 
+  public void setClaimed() {
+    isClaimed = true;
+  }
```

Comment:
We should usually make our data objects immutable to make them as predictable as possible. I don't think we should change this now since we have time constraints and the error was not made in this PR - but we should change this if we have opportunity in the future.

Reply:
agreed, this was done because of time